components/definitions/rewards.d.ts

```diff
@@ -73,7 +73,7 @@ declare namespace Rewards {
       promosDismissed?: {
         [key: string]: boolean
       }
-      walletRecoverySuccess: boolean | null
+      walletRecoverySuccess: number | null
```

Comment:
Since we're going from a boolean to a number, it would help to rename this property. Maybe `walletRecoveryStatus` or `walletRecoveryResult`?

Reply:
renamed to `walletRecoveryStatus`